server/mailrepository/deleted-messages-vault-repository/src/main/java/org/apache/james/vault/MailRepositoryDeletedMessageVault.java

```diff
@@ -124,6 +127,17 @@ public Configuration(MailRepositoryUrl urlPrefix) {
         }
     }
 
+    @Override
+    public Publisher<User> usersWithVault() {
+        return Flux.fromStream(mailRepositoryStore.getUrls()
+            .filter(url -> url.hasPrefix(configuration.urlPrefix))
```

Comment:
could be extracted to an `isVault` method

Reply:
Sorry, this is included in next commit, this is a squash mistake. Do you want me to fix it or you consider it acceptable?